components/script/dom/canvasrenderingcontext2d.rs

```diff
@@ -181,6 +181,11 @@ impl CanvasRenderingContext2DMethods for CanvasRenderingContext2D {
         self.canvas_state.restore()
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-context-2d-reset
+    fn Reset(&self) {
```

Comment:
```suggestion
    /// <https://html.spec.whatwg.org/multipage/#dom-context-2d-reset>
    fn Reset(&self) {
```

Reply:
This should likely be converted to a rustdoc comment as it's documenting a function. When doing that though, it is important not to leave raw URLs in the comment.